### DISCUSSION TITLE
希望增加在屏幕上显示中英文状态，就想大写锁定提示那样

### DISCUSSION THREAD

**Discussion by Gsy0301** [shuax/MouseInc.Settings#273] [Ideas]
玩游戏以及工作的时候因为输入法的问题，老是无法切换到中文或者英文状态，一是因为无法明显显示中文状态， 二是不知道是否正确按到切换键，所以很希望能够增加一个 可在屏幕上显示中英文状态的的功能
![Snipaste_2022-05-24_11-07-36](https://user-images.githubusercontent.com/50713061/169940824-9b280d72-1fb5-4a2f-a57d-e2a136780d08.png)
， 就想大写锁定提示那样子，非常感谢大大

**Comment by gbsix** [comment]
这个想法很高级。如果能实现这个功能，那就不用安装输入法指示器了。

**Comment by shuax** [comment]
输入法不是全局的，需要研究一下

**Reply by Gsy0301** [reply]
可以参考下这个，https://github.com/aardio/ImTip ，这个是在输入框上面显示中文状态

**Reply by gbsix** [reply]
这个做的不错，但还不是100%准确。
